minio/minio
# Add Support for Cache and S3 related metrics in Prometheus

## Description
This PR adds support for Cache related metrics

- [x] Cache Hit Count
- [x] Cache Miss Count
- [x] Data served from Cache (in Bytes)
- [x] Data received from AWS S3 (in Bytes)
- [x] Data sent to AWS S3 (in Bytes)
- [x] Number of requests sent to AWS S3

Fixes #8549

## Motivation and Context
See #8549 

## How to test this PR?
- Start MinIO Gateway mode with S3 backend
- Enable Cache
- Monitor Prometheus Scrape endpoint to see the relevant fields updating

## Types of changes

## Checklist:
